Authentication1/Authentication1/Extensions/extension+UserDefaults.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  extension+UserDefaults.swift
+//  Authentication1
+//
+//  Created by Павел Калинин on 03.07.2024.
+//
+
+import Foundation
+import UIKit
+
+extension UserDefaults {
+    func getBooks(forKey key: String) -> [Book] {
+        if let books = array(forKey: key) as? [Book.ID] {
+            print(1)
+            return books.compactMap { id in
+                BooksManager.shared.getBook(by: id)
+            }
+        }
+        return []
```

Comment:
`guard let books = array(forKey: key) as? [Book.ID] else { return []}`
`return books.compactMap { id in
                BooksManager.shared.getBook(by: id)
            }`